apps/web/screens/Regulations/RegulationsHome.tsx

```diff
@@ -0,0 +1,298 @@
+  RegulationSearchResults,
+} from '../../components/Regulations/Regulations.types'
+import { RegulationHomeTexts } from '../../components/Regulations/RegulationTexts.types'
+
+import React, { useState } from 'react'
+import { Screen } from '@island.is/web/types'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import getConfig from 'next/config'
+import { CustomNextError } from '@island.is/web/units/errors'
+import { SubpageDetailsContent } from '@island.is/web/components'
+import { RegulationsHomeImg } from '../../components/Regulations/RegulationsHomeImg'
+import { SubpageLayout } from '@island.is/web/screens/Layouts/Layouts'
+import {
+  Box,
+  Breadcrumbs,
+  Button,
+  CategoryCard,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Text,
+} from '@island.is/island-ui/core'
+import { useNamespaceStrict as useNamespace } from '@island.is/web/hooks'
+import { RegulationsSearchSection } from '../../components/Regulations/RegulationsSearchSection'
+import {
+  getParams,
+  prettyName,
+  RegulationSearchFilters,
+  useRegulationLinkResolver,
+} from '../../components/Regulations/regulationUtils'
+import { getUiTexts } from '../../components/Regulations/getUiTexts'
+import {
+  GetRegulationsSearchQuery,
+  QueryGetRegulationsSearchArgs,
+  GetRegulationsQuery,
+  QueryGetRegulationsArgs,
+  GetRegulationsYearsQuery,
+  GetRegulationsMinistriesQuery,
+  GetRegulationsLawChaptersQuery,
+  QueryGetRegulationsLawChaptersArgs,
+} from '@island.is/web/graphql/schema'
+import {
+  GET_REGULATIONS_SEARCH_QUERY,
+  GET_REGULATIONS_LAWCHAPTERS_QUERY,
+  GET_REGULATIONS_MINISTRIES_QUERY,
+  GET_REGULATIONS_QUERY,
+  GET_REGULATIONS_YEARS_QUERY,
+} from '../queries'
+
+const { publicRuntimeConfig } = getConfig()
+
+// ---------------------------------------------------------------------------
+
+type RegulationsHomeProps = {
+  regulations: RegulationSearchResults
+  texts: RegulationHomeTexts
+  searchQuery: RegulationSearchFilters
+  years: ReadonlyArray<number>
+  ministries: RegulationMinistryList
+  lawChapters: RegulationLawChapterTree
+  doSearch: boolean
+}
+
+const RegulationsHome: Screen<RegulationsHomeProps> = (props) => {
+  const { disableRegulationsPage: disablePage } = publicRuntimeConfig
+  if (disablePage === 'true') {
+    throw new CustomNextError(404, 'Not found')
+  }
+
+  const txt = useNamespace(props.texts)
+  const { linkResolver, linkToRegulation } = useRegulationLinkResolver()
+  const totalItems = props.regulations?.data?.length ?? 0
+  const stepSize = 9
+  const [showCount, setShowCount] = useState(totalItems > 18 ? stepSize : 18)
+
+  const breadCrumbs = (
+    <Box display={['none', 'none', 'block']}>
+      {/* Show when NOT a device */}
+      <Breadcrumbs
+        items={[
+          {
+            title: 'Ísland.is',
+            href: linkResolver('homepage').href,
+          },
+          /*{
+            title: 'Upplýsingasvæði',
+            href: linkResolver('article').href,
+          },*/
```

Comment:
This is just not used yet, the future plan is to have Reglugerdir and and some other pages like Stjórnartíðindi and Lögbirtingarblaðið under this Upplýsingasvæði thing. But I'll remove this for now since the other pages have been postponed.

Reply:
removed